modules/gdscript/gdscript_compiler.cpp

```diff
@@ -2670,9 +2681,7 @@ Error GDScriptCompiler::_populate_class_members(GDScript *p_script, const GDScri
 					}
 					prop_info.hint = export_info.hint;
 					prop_info.hint_string = export_info.hint_string;
-					prop_info.usage = export_info.usage | PROPERTY_USAGE_SCRIPT_VARIABLE;
-				} else {
-					prop_info.usage = PROPERTY_USAGE_SCRIPT_VARIABLE;
+					prop_info.usage |= export_info.usage;
```

Comment:
See [#81079](https://github.com/godotengine/godot/pull/81079/files#diff-43fc69f36899762bf59f1cd731dfc0be1ff0d2b648271e443944e4116c1c28a6L2673-R2683).

Reply:
Haha, nice! I'll take it out of this one!